docker/README.md

```diff
@@ -80,6 +80,63 @@ Go to https://localhost/api_keys, generate key and secret. Then run:
     # Where KEY is your openQA instance key
     # and SECRET is your openQA instance secret
 
+## Alternative: Run the Data & WebUI containers in HA mode with docker-compose
+
+    # To create the containers
+    docker-compose up -d
```

Comment:
This comment doesn't take a file from our repository or anything specific to the project as argument. I assume it must run in a certain directory then. Where would I run this command?

Reply:
@ilausuch

Reply:
I added that it should be executed in the directory openQA/docker/webui is it good engough?